docs/notes/extensions.md

```diff
@@ -1,26 +1,45 @@
 # Popular Extensions
 
+Many extensions can be found on the [Chart.js GitHub organization](https://github.com/chartjs) or on the [npm registry](https://www.npmjs.com/search?q=chartjs-plugin-).
+
+## Charts
+
+ - <a href="https://github.com/chartjs/chartjs-chart-financial" target="_blank">Financial</a> - Adds financial chart types such as a candlestick.
```

Comment:
For consistency with other entries, the name should be `chartjs-chart-financial` instead of `Financial`